docs/stats.md

```diff
@@ -590,6 +591,11 @@ and boolean expressions (e.g., {math}`(x > 0)`) are interpreted as 0/1.
   where {math}`m = \frac{1}{n}\sum_{k=1}^n x_k` with {math}`n` the total number
   of samples.
 
+`genetic_relatedness_weighted`
+: {math}`f(w_i, w_j, x_i, x_j) = \frac{1}{2}w_iw_j(x_i - m)(x_j - m)`,
+
+  where {math}`m = \frac{1}{n}\sum_{k=1}^n x_k` with {math}`n` the total number
+  of samples.
```

Comment:
Is this right? We wrote this out carefully in the theory; and looking at the C code it looks like `m` should be the average weight or something, but I've not gone back to the theory yet?